### PR TITLE
Add logo to README

### PR DESCRIPTION
**Pull request by 6gk**
Using a native font has proven to be difficult, especially when
accounting for light/dark themes - github doesn't seem to allow
inheriting font colour, and putting it outside of the svg makes the
alignment all weird.

Github also doesn't support using base64'd svgs, so I decided to put
kmonad.svg in the root directory.

Suggested by @slotThe https://github.com/kmonad/kmonad/issues/269#issuecomment-856480858

Feedback is very welcome.

### CODE REVIEW

**Comment by Vermoot** [comment]
This could be a good occasion to add links to the Discord and to IRC?

**Comment by 6gk** [comment]
Nice idea, but what should the wording be, and where should it go?

**Comment by Vermoot** [comment]
I guess for now (might change when the wiki gets a bit more fleshed out), it could just be an addition to the introduction (so it gets good visibility), something like:

Additionally, if you need any help or if you just want to say hi, you can join our [Discord server](https://discord.gg/3tFfWmnahN) or jump into our [IRC channel](https://kiwiirc.com/nextclient/irc.freenode.net/kmonad). We're friendly folk!

Btw I really like this version of the logo, with the full name. The uppercase M feels *maybe* a little off, idk if you've tried with a lowercase one. Might be just right as is, though.

**Comment by 6gk** [comment]
> M feels maybe a little off

I had this feeling too, kinda.

I didn't really experiment with it too much tbh

**Comment by 6gk** [comment]
image

I had this idea, but I can't get the execution down right.

**Comment by 6gk** [comment]
image

[sorry for spam]

What about small caps?

**Comment by Vermoot** [comment]
I do think I prefer lowercase M, it looks more like a single word.

I don't like the doubled M though, it's too messy IMO. Just having the K as the logo and then completing the name looks better.

I can't open the latest one.

**Comment by 6gk** [comment]
> I do think I prefer lowercase M

Personally I'm not a fan of the way lowercase m looks tbh - in most fonts, not just this one.

> it looks more like a single word.

Fair point.

**Comment by slotThe** [comment]
On Tue, Jun 08 2021 02:52, Vermoot wrote:
> I do think I prefer lowercase M, it looks more like a single word.

kmonad's name is inspired by xmonad (who would have thought :)) and
there we (e.g. in the module names) very often use and see XMonad.  From
that perspective I think KMonad reads much nicer than Kmonad

**Comment by Vermoot** [comment]
Fair point, uppercase M it is then I guess

**Comment by 6gk** [comment]
image

What about all-caps?

**Comment by Vermoot** [comment]
Eh, I prefer KMonad. Seems more uh, human

File: README.md [hunk 1]
```diff
@@ -20,6 +17,8 @@ KMonad is an advanced tool that lets you infinitely customize and extend the fun
 
 If you want to get started with the latest, stable binary release, please check out the master branch, if you are interested in the latest additions and tweaks, switch on over to develop and compile your own binary.
 
+Additionally, if you need any help or just want to say hi, you can join our [Discord server](https://discord.gg/3tFfWmnahN) or jump into our [IRC channel](https://web.libera.chat/#kmonad) (#kmonad@irc.libera.chat), which is also bridged with matrix (#kmonad:libera.chat).
+
```
[the remark this answers is not in the document]
Comment (6gk) [reply README.md]: Good catch!  Thanks.

**Comment by slotThe** [comment]
Thanks!

**Comment by 6gk** [comment]
Thank you too!

**Comment by david-janssen** [comment]
Thanks indeed!!

**Pull request merged**